docs/source/revision_history.rst

```diff
@@ -3,6 +3,12 @@ Revision history
 
 List of major changes:
 
+- Upcoming
+    * Publish ``mypy`` version 0.550 on PyPI.
+
+      * We soon intend to drop support for Python 3.3, which has reached end-of-life. Please comment on `#4036 <https://github.com/python/mypy/issues/4036>`_ if this is an issue.
```

Comment:
This should just be "Running mypy now requires Python 3.4 or higher." Perhaps also add "However Python 3.2 and 3.3 are still valid for the target of the analysis (i.e. the `--python-version` flag)."

We already announced we would drop 3.3 support for running mypy in the blog post for 0.530 -- we just never followed through. FWIW the text there was:

> Note that future mypy versions will still support _checking_ older Python versions — they just won’t _run_ on 3.3 or earlier. It’s easy to have multiple Python versions installed, so this won’t prevent anyone from running mypy.

Reply:
Updated!